### PR TITLE
improve the plugin for integration

### PR DESCRIPTION
**Pull request by valerii-zinchenko**
Here are the updates I with @sasha240100 made after I tried to connect this plugin to our project.

### CODE REVIEW

**Comment by musculman** [comment]
Can you please bump the version of the package as well?

**Comment by valerii-zinchenko** [comment]
> Can you please bump the version of the package as well?

I updated to 0.1.1

File: src/index.ts [hunk 1]
```diff
@@ -139,19 +137,22 @@ export default class HarpGL extends Layer {
...
+        });
 
-        canvas.style.width = '100%';
-        canvas.style.height = '100%';
+        this.m_glContainer.appendChild(canvas);
 
         this.m_mapView = new MapView({
             canvas,
             alpha: false,
             pixelRatio: 1,
```
Comment (sasha240100) [inline src/index.ts]: Please also remove `pixelRatio: 1` as was discussed

File: src/index.ts [hunk 2]
```diff
@@ -139,19 +137,22 @@ export default class HarpGL extends Layer {
...
+            height: "100%"
+        });
 
-        canvas.style.width = '100%';
-        canvas.style.height = '100%';
+        this.m_glContainer.appendChild(canvas);
 
         this.m_mapView = new MapView({
             canvas,
             alpha: false,
```
Comment (sasha240100) [inline src/index.ts]: Default is `false`, can be removed

File: src/index.ts [hunk 3]
```diff
@@ -9,9 +9,9 @@ import bezier from 'bezier-easing';
 import {DomUtil, LatLng, Layer, LayerOptions} from 'leaflet';
 import './draggable-patch';
 
-interface HarpLeafletOptions extends LayerOptions, MapViewOptions {
+type HarpLeafletOptions = Omit<LayerOptions & MapViewOptions, "canvas"> & {
     [key: string]: any;
```
Comment (sasha240100) [inline src/index.ts]: `[key: string]: any;` is no longer needed as we rely completely on `LayerOptions & MapViewOptions`

File: src/index.ts (hunk 1 above)
Comment (valerii-zinchenko) [reply src/index.ts]: done

File: src/index.ts (hunk 2 above)
Comment (valerii-zinchenko) [reply src/index.ts]: done

File: src/index.ts (hunk 3 above)
Comment (valerii-zinchenko) [reply src/index.ts]: done

**Comment by musculman** [comment]
If @sasha240100 is fine with it, we can merge and release a new version of the npm package. From my side it looks good

**Pull request merged**